### PR TITLE
docs: introduce version 2 of the management API 

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Issues: https://tracker.mender.io/browse/MEN-1981

Please take a look really carefully.
There is no GET /devices/{id}/auth/{aid} endpoint in the new API since I can't see a point in having this.

### CODE REVIEW

File: docs/management_api_v2.yml [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+
+basePath: '/api/management/v2/devauth/'
+host: 'mender-device-auth:8080'
+schemes:
+  - http
+
+paths:
+  /devices:
+    get:
+      summary: Get a list of tenant's devices.
```
Comment (michaelclelland) [inline docs/management_api_v2.yml]: Is the list sorted? It might be useful to sort by created_ts or updated_ts

File: docs/management_api_v2.yml [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+      description: |
+        Provides a list of tenant's devices, with optional device status filter.
```
Comment (michaelatmender) [inline docs/management_api_v2.yml]: Is this list sorted? It might be useful to sort by created_ts or updated_ts

Comment (kjaskiewiczz) [reply docs/management_api_v2.yml]: currently, in the deviceauth and deviceadm services, we are sorting devices by id
to me, it sounds reasonable to sort them by created_ts
@mchalski @maciejmrowiec what do you think?

Comment (maciejmrowiec) [reply docs/management_api_v2.yml]: 👍

File: docs/management_api_v2.yml [hunk 3]
```diff
@@ -0,0 +1,507 @@
...
+    type: object
+    properties:
+      error:
+        description: Description of the error
+        type: string
+  PreAuthSet:
+    type: object
+    properties:
+      device_identity:
+        type: string
```
[the remark this answers is not in the document]
Comment (kjaskiewiczz) [reply docs/management_api_v2.yml]: good idea

**Comment by kjaskiewiczz** [comment]
Comments addressed by adding two new commits.

**Pull request merged**